compiler/src/yul/base.rs

```diff
@@ -0,0 +1,23 @@
+use yultsur::yul;
```

Comment:
what do you think about calling this module `base_types` instead of just `base`?